### ISSUE TITLE
'VPN is on/off' static text not being read by the iOS VoiceOver screen reader

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7776]
[~accountid:60acc5180c6f2d0068633c71] reported that the 'VPN is on/off' static text in the Home page is not being read by the iOS VoiceOver screen reader. It is read on Windows and Android.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5394)
┆Reporter: Vinoo Cherian

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

* Does this mean that users needing this feature can’t know if the VPN is ON or OFF?
  * Is this new to 2.17 or always been the case?

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Chris Rooney , Santiago Andrigo ,

Matt Lichtenstein tried this on iOS and can’t repro. Seems to work fine. See attached:

[^RPReplay_Final1695665811.MP4]

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

Cannot repro on iOS

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
